pkg/apply/applier.go

```diff
@@ -188,6 +185,65 @@ func splitInfos(infos []*resource.Info) ([]*resource.Info, []*resource.Info) {
 	return resources, groupingObjectTemplates
 }
 
+// buildTaskQueue takes the slice of infos and object identifiers, and
```

Comment:
The `infos` and the `identifiers` both represent objects (or resources). Can we put more information in this comment on these parameters and which objects they represent?

Reply:
I've added additional comments here. It is a bit awkward that we need to work with both the infos and the identifiers, but I can't see any obvious ways to fix this. We should keep looking for ways to simplify this.